doc/cephadm/troubleshooting.rst

```diff
@@ -139,39 +144,41 @@ To fetch all log files of all daemons on a given host, run::
 Collecting Systemd Status
 -------------------------
 
-To print the state of a systemd unit, run::
+To print the state of a systemd unit, run a command of the following form: 
 
-      systemctl status "ceph-$(cephadm shell ceph fsid)@<service name>.service";
+.. prompt:: bash
 
+   systemctl status "ceph-$(cephadm shell ceph fsid)@<service name>.service";
 
-To fetch all state of all daemons of a given host, run::
 
-    fsid="$(cephadm shell ceph fsid)"
-    for name in $(cephadm ls | jq -r '.[].name') ; do
-      systemctl status "ceph-$fsid@$name.service" > $name;
-    done
+To fetch all state of all daemons of a given host, run the following shell
```

Comment:
s/all/the/

Reply:
Oh, good God. How did I miss that?

Accepted.

Reply:
Arguably subtle.  I had to read the context carefully to decide if it was in fact correct. There could be command variants that show all state of some daemons or some state of all daemons etc.

Reply:
I think it was just a matter of reading over 300 lines of text for a couple of hours, until the text became so stale that I needed a second pair of eyes.

Reply:
Fortunately I have two of those.